client/src/pages/EquipmentBrowser/EquipmentBrowser.tsx

```diff
@@ -1,22 +1,85 @@
 import React from 'react';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import 'react-tabs/style/react-tabs.css';
 import styles from './EquipmentBrowser.module.css';
 
+export interface Item {
+  name: string;
+  slug: string;
+  category: {
+    name: string;
+    slug: string;
+    id: number;
+  };
+  total_count: number;
+  image: string | null;
+}
+
 const EquipmentBrowser: React.FC = () => {
+  const [items, setItems] = React.useState<Array<Item>>([]);
+  const listItems = items.map(item => (
+    <div className={styles.equipment} key={item.name}>
+      <div>
+        <img src="https://via.placeholder.com/150" />
+      </div>
+      <div>{item.name}</div>
+    </div>
+  ));
+  React.useEffect(() => {
+    fetch('https://tutv-mock.now.sh/api/v1/equipment/')
+      .then(response => response.json())
+      .then(response => setItems(response.data))
+      .catch(error => console.error(error));
+  }, []);
   return (
-    <div className={styles.header}>
-      <div className={styles.timepicker}>
-        <div>
-          <div>Checkout Time</div>
-          <input type="datetime-local" />
+    <div>
+      <div className={styles.header}>
+        <div className={styles.timepicker}>
+          <div>
+            <div>Checkout Time</div>
+            <input type="datetime-local" />
+          </div>
+          <div>
+            <div>Return Time</div>
+            <input type="datetime-local" />
+          </div>
         </div>
         <div>
-          <div>Return Time</div>
-          <input type="datetime-local" />
+          <button>Cancel Request</button>
```

Comment:
Another note - these should probably use our `<Button>` component instead of being plain HTML `<button>`s